test/unit/lib/plugins/aws/package/compile/events/kafka.test.js

```diff
@@ -0,0 +1,453 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../utils/run-serverless');
+
+const { expect } = chai;
+
+chai.use(require('chai-as-promised'));
+
+describe('AwsCompileKafkaEvents', () => {
```

Comment:
I'm not sure if this is documented, but it's recommended to name new describe blocks after the name of the test file, so in this case it will be `test/unit/lib/plugins/aws/package/compile/events/kafka.test.js`